### DISCUSSION TITLE
Network baseline with Security Onion?

### DISCUSSION THREAD

**Discussion by glwallum** [Security-Onion-Solutions/securityonion#4568] [Unsupported Versions]
Hello

I am looking at achieving network baseline information using Zeek logs. Does anyone have any experience with this?

I found a few options in bropy (https://github.com/hashtagcyber/bropy) and netbase (https://github.com/DynamiteAI/netbase) but looking for some more information

Thank you

**Comment by weslambert** [comment]
I would wait at least a few weeks after initial collection, then try agging/stacking Zeek records in Hunt or Kibana to get a lay of the land (obviously you can do this as you go and don't have to wait, but from a higher-level perspective, it's good to be able to see over a greater timespan). It can take time to be able to take these kinds of measurements and really understand what is considered normal.
